### PR TITLE
Fix/1571/count of application new

### PR DESCRIPTION
**Pull request by aquibmoin786**
#1571

### Description

- We are added new popup icon in requirement field. This popup icon works to calculate round wise applicant.

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by P4NK4J — CHANGES_REQUESTED** [review]
@aquibmoin786 Shared some testing feedback with you, on Google chat. Please check and let me know if you have any questions.

File: resources/sass/_utilities.scss [hunk 1]
```diff
@@ -22,7 +22,7 @@
 .overflow-y-scroll {
 	overflow-y: scroll;
 }
-.c-pointer {
+.c-pointer,.nav-item{
```
Comment (GauravGusain98) [inline resources/sass/_utilities.scss:25]: Why we are giving this styel to `.nav-item` we can use `c-pointer` class directly to apply these styles

File: resources/views/hr/menu.blade.php [hunk 2]
```diff
@@ -14,4 +14,8 @@
     <li class="nav-item">
         <a class="nav-item nav-link {{ Request::is('hr/recruitment/campaigns*') ? 'active' : '' }}" href="{{ route('recruitment.campaigns') }}"><i class="fa fa-envelope"></i>&nbsp;Campaigns</a>
     </li>
+    </li>
+    <li class="nav-item" data-toggle="modal" data-target="#countIcon">
```
Comment (GauravGusain98) [inline resources/views/hr/menu.blade.php:18]:
```suggestion
    <li class="nav-item c-pointer" data-toggle="modal" data-target="#countIcon">
```

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment

File: resources/views/hr/menu.blade.php (hunk 2 above)
Comment (aquibmoin786) [reply resources/views/hr/menu.blade.php:18]: I will fixing it

File: resources/views/hr/application/total-application-count-modal.blade.php [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="totalCount">Total Open Applications</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        {{-- @dd($rounds) --}}
+      @foreach ($rounds as $key=>$round)
```
Comment (GauravGusain98) [inline resources/views/hr/application/total-application-count-modal.blade.php:13]:
```suggestion
          @foreach ($rounds as $key=>$round)
```

File: resources/sass/_utilities.scss [hunk 4]
```diff
@@ -22,7 +22,7 @@
 .overflow-y-scroll {
 	overflow-y: scroll;
 }
-.c-pointer {
+.c-pointer{
```
Comment (GauravGusain98) [inline resources/sass/_utilities.scss:25]:
```suggestion
.c-pointer {
```

File: resources/views/hr/application/total-application-count-modal.blade.php [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+  <div class="modal-dialog" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <h5 class="modal-title" id="totalCount">Total Open Applications</h5>
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+          <span aria-hidden="true">&times;</span>
+        </button>
+      </div>
+      <div class="modal-body">
+        {{-- @dd($rounds) --}}
```
Comment (GauravGusain98) [inline resources/views/hr/application/total-application-count-modal.blade.php:12]: remove this dev code

File: Modules/HR/Http/Controllers/Recruitment/ApplicationController.php [hunk 6]
```diff
@@ -27,7 +27,6 @@ abstract class ApplicationController extends Controller
     abstract public function getApplicationType();
 
     protected $service;
-
```
Comment (GauravGusain98) [inline Modules/HR/Http/Controllers/Recruitment/ApplicationController.php:30]: Please add a new line here

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comment

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Review by P4NK4J — APPROVED** [review]
Looks good.

**Pull request merged**